Casks/w/windsurf.rb

```diff
@@ -0,0 +1,38 @@
+cask "windsurf" do
+  arch arm: "darwin-arm64", intel: "darwin-x64"
```

Comment:
```suggestion
  arch arm: "arm64", intel: "x64"
```

This change allows for more flexibility in the future, as many apps also use short identifiers, e.g., in API responses.

Reply:
Thanks for catching this! I've updated the arch identifiers to use the shorter format:
- Changed "darwin-arm64" to "arm64"
- Changed "darwin-x64" to "x64"

This indeed provides more flexibility and aligns better with common API response formats.